### DISCUSSION TITLE
Thycotic.secretserver v0.60.6 not in PSGallery.

### DISCUSSION THREAD

**Discussion by MaximoTrinidad** [thycotic-ps/thycotic.secretserver#266] [General]
Hi There,

Since the VPV Secret Server upgrade to 11.1 broke some of the essentials module functions  (... check-in/check-out ...). I've been waiting for an update. version.   I notice it has not been updated in the PowerShell Gallery repo.

Any ETA on it?

**Comment by tylerezimmerman** [comment]
Apologies about the delay. Due to some outstanding issues, we are unable to post the fix to the PowerShell Gallery repo at this time. We recommend that you take the most recent commit and use that to build the solution locally.

**Comment by EndlessTrax** [comment]
The latest patch release is now available in the PowerShell Gallery - https://www.powershellgallery.com/packages/Thycotic.SecretServer/0.60.6
